File: apps/mybookkeeper/frontend/src/shared/types/welcome-manual/welcome-manual-place-response.ts
/**
 * Mirrors backend `PlaceResponse`. A place is a restaurant recommendation
 * parented directly to the manual (not to a section) — guests browse the
 * flat list via the "Where to Eat" directory.
 */
export interface WelcomeManualPlaceResponse {
  id: string;
  manual_id: string;
  name: string;
  cuisine: string;
  price_tier: "$" | "$$" | "$$$" | null;
  note: string | null;
  map_url: string | null;
  display_order: number;
  created_at: string;
}

File: apps/mybookkeeper/backend/app/schemas/welcome_manuals/welcome_manual_place_response.py
import uuid
from datetime import datetime

from pydantic import BaseModel, ConfigDict


class WelcomeManualPlaceResponse(BaseModel):
    id: uuid.UUID
    manual_id: uuid.UUID
    name: str
    cuisine: str
    price_tier: str | None = None
    note: str | None = None
    map_url: str | None = None
    display_order: int
    created_at: datetime

    model_config = ConfigDict(from_attributes=True)
